### ISSUE TITLE
Webpack doesn't report plugin failure

### ISSUE THREAD

**Issue by yrtimiD** [TypeStrong/fork-ts-checker-webpack-plugin#167]
At the moment I enable ForkTsCheckerWebpackPlugin plugin it starts to fail with 
CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory

But the real problem is: webpack doesn't recognizes that plugin failed and reports exit code 0.

part of log looks next:
```
> webpack

Starting type checking service...
Using 1 worker with 8192MB memory limit
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
npm verb lifecycle angular-examplecomponent@1.0.0~build: unsafe-perm in lifecycle true
npm info lifecycle angular-examplecomponent@1.0.0~postbuild: angular-examplecomponent@1.0.0
npm verb exit [ 0, true ]
npm timing npm Completed in 8078ms
npm info ok
```

So, either plugin doesn't report failure, or webpack doesn't get it.

webpack configuration is pretty basic:
```javascript
//partial
var ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
var config = {
		rules: [{
			test: /\.ts$/,
			use: [{
				loader: "ts-loader",
				options: {
					allowTsInNodeModules: true,
					transpileOnly: true
				}
			}]
		}],
	plugins: [
		new ForkTsCheckerWebpackPlugin({
			memoryLimit: 8192
		})
	]
}

```

**Comment by ghost** [comment]
@yrtimiD Have you found a solution? I am facing the same problem. 

In production mode, the build falis. So I have tried adding NoEmitOnErrorsPlugin (what production mode does) but it did not work. When using custom plugin just to debug and print the compilation.erros array, its empty. In production mode, in contains the errors.

**Comment by johnnyreilly** [comment]
Have you tried `async: false`? The plugin deliberately doesn't report errors to webpack unless this is set for speed reasons

**Comment by ghost** [comment]
@johnnyreilly Thank you. I have put it in the wrong place (ts-loader) and now its working. I get the errors array filled with the errors.

**Comment by johnnyreilly** [comment]
Nice!
